osmomath/math.go

```diff
@@ -203,3 +206,62 @@ func OrderOfMagnitude(num Dec) int {
 
 	return order
 }
+
+var legacyPrecisionOf5sReuse = new(big.Int).Exp(big.NewInt(5), big.NewInt(sdkmath.LegacyPrecision), nil)
+var twoFactorsToRemovePerStep = uint(sdkmath.LegacyPrecision) - 1
+var divisorsNoPowersOfTwo = []*big.Int{nil}
```

Comment:
Did you mean to actually add a nil big.Int as the first element or were you trying to initialize the slice as nil? I ask because down below you invoke append, yet have nil as the first element.

Reply:
I intended to have nil as the first element, and never 0-index this